packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -358,6 +358,7 @@ export type CallCompositeIcons = {
     ParticipantItemScreenShareStart?: JSX.Element;
     VideoTileMicOff?: JSX.Element;
     LocalCameraSwitch?: JSX.Element;
+    InvalidRoom?: JSX.Element;
```

Comment:
Looks like you went a bit too far with removal of conditional compilation.

This icon shouldn't be there in the stable API.